Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM node:16-slim AS builder
+
+RUN apt-get update && apt-get install -y tree
+
+WORKDIR /app
+
+# add the dependencies
+ADD package.json yarn.lock /app/
+RUN yarn install --frozen-lockfile
+
+# add the remaining parts of the produce the build
+COPY . /app
+RUN yarn codegen && yarn build
+
+FROM onfinality/subql-node-cosmos:v0.2.0
+
+# add extra tools that are required
+ADD https://github.com/mikefarah/yq/releases/download/v4.26.1/yq_linux_amd64 /usr/local/bin/yq
```

Comment:
I was not previously aware of this tool :star_struck: - thanks for sharing @ejfitzgerald!